cmd/helm/install.go

```diff
@@ -285,12 +286,32 @@ func (v *values) Set(data string) error {
 	return nil
 }
 
+func typedVal(val string) interface{} {
+	if val == "true" {
```

Comment:
However, I'm also okay enforcing people to use `true` and `false` so we don't need to worry about order issues.